### DISCUSSION TITLE
got code 9991 with msg "Missing or invalid parameters" Error

### DISCUSSION THREAD

**Discussion by redDwarf03** [phantom/docs#235] [Help & Support]
### Summary

Hello

When we send a transaction without value property filled, Phantom wallet send an RPC error "got code 9991 with msg "Missing or invalid parameters" 
But i use eth_sendTransaction with a Transaction.value = null to create a HTLC Contract

Is it a desired behavior to generate an error in this case, knowing that other EVM wallets do not return an error if the value is not filled in?

Thx

### Steps to Reproduce

Create for example a transaction: 

eth_sendTransaction
from: 0xb476bf84be229f0eb0ef94402eb265ee2c4dffe2
to: 0xd5ca9f76495b853a5054814a10b6365ee8ed745b
gasPrice: 0xad7e673ab
gas: 0x16e360
value: null
data: 0x2064295e0029d025910906909f90af582c2b49eb5ae39efa7f60d1fca9fcf0c508712c110000000000000000000000000000000000000000000000008ac7230489e80000

### Phantom Version

24.8.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hi @redDwarf03,

Thanks for writing in!

Do you have a version of this hosted somewhere that we can reproduce for investigation?

**Reply by redDwarf03** [reply]
Hi. Yes. I've developed a bridge between EVM Blockchain and Archethic Blockchain.

You can find the testnet here: https://bridge.testnet.archethic.net/
You can use Sepolia network to test it

You can find the documentation of the bridge here: https://wiki.archethic.net/participate/bridge/ and the tuto: https://wiki.archethic.net/participate/bridge/usage/bridge-front
And a dynamic tuto to install the Archethic Wallet: https://wiki.archethic.net/participate/DeFi/ or https://wiki.archethic.net/FAQ/wallet#-how-to-create-a-wallet-on-the-archethic-blockchain


The bridge works for users who using Metamask, Frame, Rabby but they create issues about Phantom.
The bridge use Flutter framework and a web3 flutter lib to interact with EVM Wallet https://pub.dev/packages/webthree
After analysis, i saw that the value property is the cause of the Phantom issue.

I hope these info can help you

**Reply by hillieton** [reply]
I have the exact same issue and my computer is broken now so I can’t un nullify the certificate but when I try to transfer or exchange your account number it comes back as error as well
